climpred/tests/conftest.py

```diff
@@ -373,6 +423,15 @@ def PM_ds_control_1d_ym_cftime(PM_ds_control_1d):
     return PM_ds_control_1d
 
 
+@pytest.fixture
+def perfectModelEnsemble_initialized_control_1d_ym_cftime(
+    PM_ds_initialized_1d_ym_cftime, PM_ds_control_1d_ym_cftime
+):
```

Comment:
Add description

Reply:
I know it's kinda in the name, but it gets overwhelming when there's so many in `conftest`. I think it's helpful